oktoberfest/runner.py

```diff
@@ -127,15 +126,15 @@ def _annotate_and_get_library(spectra_file: Path, config: Config, tims_meta_file
         search = pp.load_search(config.output / "msms" / spectra_file.with_suffix(".rescore").name)
         library = pp.merge_spectra_and_peptides(spectra, search)
         pp.annotate_spectral_library(library, mass_tol=config.mass_tolerance, unit_mass_tol=config.unit_mass_tolerance)
-        library.write_as_hdf5(hdf5_path).join()  # write_metadata_annotation
+        library.write_as_hdf5(hdf5_path, True).join()  # write_metadata_annotation
```

Comment:
don't forget to change this, using the anndata write oneliner, this is simply write_as_hdf5(<path>)